### ISSUE TITLE
Playing on Large devices

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#160]
```
There has been talk about being able to play on larger devices and images scale 
correctly. A bug popped up over on the forums on a 10" tablet and I assume 
other tablets around the same size that the text does not scale very well and 
the scroll bar is small so it is also hard to scroll.


I am giving this a medium priority with Honeycomb and the Moto Xoom coming out 
in Q1 2011. I only expect more and more tablets to come out.
```

Original issue reported on code.google.com by `sdeva...@gmail.com` on 13 Feb 2011 at 5:04

**Comment by GoogleCodeExporter** [comment]
```
I play on both my phone and my 10" tablet. It scales extremely well on the 
tablet (viewsonic gtablet running  vegan mod). The only issue ,as you noted, is 
the scrolling on dialog boxes. You can barely read text and you have to try to 
scroll to read it.
```

Original comment by `steve.wa...@gmail.com` on 14 Feb 2011 at 2:39

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Adding in issue 161 to this issue, both are issues with playing on larger 
devices.
```

Original comment by `sdeva...@gmail.com` on 15 Feb 2011 at 1:34

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 24 Feb 2011 at 10:05
- Added labels: **Milestone-0.6.9**

**Comment by GoogleCodeExporter** [comment]
```
This is ready for 0.6.9 since r55.
```

Original comment by `oskar.wi...@gmail.com` on 7 Apr 2011 at 10:00
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:50
- Changed state: **Done**
